src/librustc/build.rs

```diff
@@ -8,8 +8,14 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+use std::env;
+
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     println!("cargo:rerun-if-env-changed=CFG_LIBDIR_RELATIVE");
     println!("cargo:rerun-if-env-changed=CFG_COMPILER_HOST_TRIPLE");
+
+    if env::var_os("RUSTC_VERIFY_LLVM_IR").is_some() {
```

Comment:
You'll want to also add a rerun if env changed directive here.

Reply:
Done!